src/middlewares/VerifyUserStatus.js

```diff
@@ -0,0 +1,32 @@
+import Response from '../helpers/response';
```

Comment:
I think we should combine this method/function with the one that verifies the jwt. That way, we'll check if the jwt is valid, if the user exists and if the user is active.

Reply:
I will look into it